azure-vnet/main.tf

```diff
@@ -0,0 +1,17 @@
+provider "azurerm" {
+  features {}
+  }
+
+resource "azurerm_virtual_network" "test" {
+  name                = var.vnet_name
+  address_space       = var.vnet_address_space
+  resource_group_name = var.resource_group_name
+  location            = var.location
+  }
+
+resource "azurerm_subnet" "test" {
```

Comment:
we'll need to create multiple subnets for this vnet (one for the k8s and another for the VMs, for example). How can we declare this resource in a way that allows us to create multiple subnets only by editing the terragrunt.hcl file?